docs/api/app.md

```diff
@@ -782,27 +790,38 @@ starts:
 const {app} = require('electron')
 let myWindow = null
 
-const isSecondInstance = app.makeSingleInstance((commandLine, workingDirectory) => {
+const gotTheLock = app.requestSingleInstanceLock()
+
+app.on('second-instance', (commandLine, workingDirectory) => {
   // Someone tried to run a second instance, we should focus our window.
   if (myWindow) {
     if (myWindow.isMinimized()) myWindow.restore()
     myWindow.focus()
   }
 })
 
-if (isSecondInstance) {
+if (!gotTheLock) {
   app.quit()
+} else {
+  // Create myWindow, load the rest of the app, etc...
```

Comment:
Non-primary apps shouldn't register for 'second-instance' events, so the `app.on('second-instance'` should probably go here. Devs are going to copy-paste this, so let's make it :100: